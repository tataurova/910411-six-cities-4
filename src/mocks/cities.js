export const cities = [
  `Paris`,
  `Cologne`,
  `Brussels`,
  `Amsterdam`,
  `Hamburg`,
  `Dusseldorf`,
];
