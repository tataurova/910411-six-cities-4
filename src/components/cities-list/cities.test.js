import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list.jsx";
import cities from "../../mocks/cities.js";

const activeCity = `Paris`;

describe(`<CitiesList />`, () => {
  it(`Render App`, () => {
    const tree = renderer
      .create(<CitiesList
        cities={cities}
        activeCity={activeCity}
        onMenuClick={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
