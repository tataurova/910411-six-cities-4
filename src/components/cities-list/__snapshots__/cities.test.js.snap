// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesList /> Render App 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span
        data-city="Paris"
        onClick={[Function]}
      >
        Paris
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        data-city="Cologne"
        onClick={[Function]}
      >
        Cologne
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        data-city="Brussels"
        onClick={[Function]}
      >
        Brussels
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        data-city="Amsterdam"
        onClick={[Function]}
      >
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        data-city="Hamburg"
        onClick={[Function]}
      >
        Hamburg
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        data-city="Dusseldorf"
        onClick={[Function]}
      >
        Dusseldorf
      </span>
    </a>
  </li>
</ul>
`;
